CONTRIBUTING.md

```diff
@@ -22,7 +22,16 @@ Repository | Component | Description | Devfile | Documentation
 ---| [tests](https://github.com/eclipse/che/tree/master/tests) | source code of our integration tests. | | |
 [che-theia](https://github.com/eclipse-che/che-theia) | | Theia IDE integrated in Che. | [devfile](https://github.com/eclipse-che/che-theia/blob/main/devfiles/che-theia-all.devfile.yaml) | [doc](https://github.com/eclipse-che/che-theia/blob/main/CONTRIBUTING.md)
 ---| [generator](https://github.com/eclipse-che/che-theia/tree/main/generator) | `che:theia init` CLI to prepare and build che-theia | | |
-[chectl](https://github.com/che-incubator/chectl) | | The CLI to install Che, create and start workspaces and devfiles | [devfile](https://github.com/che-incubator/chectl/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/chectl/blob/master/CONTRIBUTING.md)
+[chectl](https://github.com/che-incubator/chectl
+https://github.com/che-incubator/che-code) | | The CLI to install Che, create and start workspaces and devfiles
```

Comment:
`The CLI to install Che ...` - is something wrong here? :slightly_smiling_face:

Reply:
yes the script failed due to a typo in the yaml file

Reply:
should be good now 👍 
thanks for the report

Reply:
no problem )